pootle/static/js/common.js

```diff
@@ -203,6 +205,16 @@ PTL.common = {
       });
     });
 
+
+    /* Bind hotkeys */
+    const hotkeys = mousetrap(document.body);
+
+    hotkeys.bind('f1', (e) => {
```

Comment:
One should be able to invoke keyboard help dialog while being inside the input box (which is what users will have by default in translation mode). So bare <kbd>?</kbd> is not an option. We can try <kbd>⌘</kbd>+<kbd>?</kbd> or <kbd>Control</kbd>+<kbd>?</kbd> or whatever works on Mac and doesn't clash with input fields.